tools/ogs-tools/AssignMaterialParameter.xml

```diff
@@ -0,0 +1,50 @@
+<tool id="assign_material_parameter" name="Assign Material Parameter" version="0.1.0+galaxy0" python_template_version="3.5" profile="21.05">
```

Comment:
If possible make the tool version match the version of the actual tool (and append `+galaxyX`). Also do it with a macro token. See [here](https://github.com/galaxyproject/tools-iuc/tree/main/tools/dada2) for inspiration.

Reply:
`python_template_version` can be removed.

Reply:
Use a more recent profile, e.g. 23.0